Process.json

```diff
@@ -7,4 +7,9 @@
 	"project_name": "joao-paulo-parity",
 	"owner": "joao-paulo-parity",
 	"matrix_room_id": "!VDLSjvBpNqqgyMDIBO:matrix.parity.io"
+},
+{
+	"project_name": "bot-playground",
```

Comment:
This project does not exist in this repository

Reply:
Can we document what this file is for then maybe?